go/ct/rlz/domains.go

```diff
@@ -138,6 +138,35 @@ func (d u256Domain) SamplesForAll(as []U256) []U256 {
 	return res
 }
 
+////////////////////////////////////////////////////////////
+// Address
+
+type addressDomain struct{}
+
+func (addressDomain) Equal(a tosca.Address, b tosca.Address) bool { return a == b }
+func (addressDomain) Less(a tosca.Address, b tosca.Address) bool  { panic("not useful") }
+func (addressDomain) Predecessor(a tosca.Address) tosca.Address   { panic("not useful") }
+func (addressDomain) Successor(a tosca.Address) tosca.Address     { panic("not useful") }
```

Comment:
`SamplesForAll` calls this function, so it should be implemented. It could just increment the address, this would also be a simple implementation for `Predecessor`.

Reply:
Actually, `SamplesForAll` is not needed either. I removed its implementation.

I would not want to implement a predecessor and successor for addresses just for this now, where it is not needed.